syntax = "proto2";


package protos;


enum Activation {
    NONE = 0;
    RELU = 1;
    SIGMOID = 2;
    ELU = 3;
}

enum Padding {
    valid = 1;
    same = 2;
}

enum Initializer {
    ones_initializer = 0;
    variance_scaling_initializer = 1;
}

enum Optimizer {
    AdamOptimizer = 0;
    GradientDescentOptimizer = 1;
}
